src/lib/datepicker/datepicker.spec.ts

```diff
@@ -223,6 +223,14 @@ describe('MdDatepicker', () => {
         expect(ownedElement).not.toBeNull();
         expect((ownedElement as Element).tagName.toLowerCase()).toBe('md-calendar');
       });
+
+      it('should throw when given wrong data type', () => {
+        testComponent.date = '1/1/2017' as any;
+
+        expect(() => fixture.detectChanges()).toThrow();
```

Comment:
Nit: Add a regex to make this clear which error should surface? e.g. `toThrowError(new RegExp(...)`